app/controllers/users_controller.rb

```diff
@@ -13,9 +18,45 @@ def create
     end
   end
 
+  # GET /users/1/edit
+  def edit
+    set_user
+  end
+
+  # PATCH/PUT /users/1
+  def update
+    set_user
+    respond_to do |f|
+      if @user.update(user_params)
+        f.html do
+          redirect_to @user, notice: 'User was successfully updated.'
+        end
+      else
+        f.html { render :edit }
+        f.json { render json: @user.errors, status: :unprocessable_entity }
+      end
+    end
+  end
```

Comment:
And also make the update method look like this.
` def update`
`    @user = User.find(params[:id])`
`    if @user.update(user_params)`
 `     redirect_to new_user_path`
   ` else`
    `  render :edit`
    `end`
 ` end`

Reply:
Implementation done.